src/Input/Select/Select.tsx

```diff
@@ -197,6 +197,14 @@ class Select extends React.Component<Props, State> {
     } else if (e.keyCode === 13) {
       inputElement.blur();
       const activeElement = this.getActiveElement();
+      const activeElementIndex = Number(get(activeElement, 'dataset.id'));
+      const activeChild =
+        isInteger(activeElementIndex) &&
+        React.Children.toArray(children).find(
+          (child, i) => i === activeElementIndex
```

Comment:
I think we can simply use `React.Children.toArray(children)[activeElementIndex]` instead.